buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt

```diff
@@ -69,6 +69,7 @@ private const val TRUTH_VERSION = "1.1"
 private const val WORK_VERSION = "2.4.0"
 private const val FLIPPER_VERSION = "0.63.0"
 private const val FLIPPER_SO_LOADER_VERSION= "0.9.0"
+private const val SHIMMER_VERSION= "0.5.0"
```

Comment:
Please add this alphabetically to the list (and move the Flipper version, if we're already making changes here)